Developers/Unity Project/Spooksville/Assets/Scripts/Items/WeaponManager.cs

```diff
@@ -7,12 +7,7 @@ public class WeaponManager : MonoBehaviour
     public static WeaponManager instance;
     private List<Weapon> weapons = new List<Weapon>();
```

Comment:
If you make this a `public Weapon[] weapons` and make `Weapon` serializable I think you'll be able to edit all the weapon stats in the editor instead of having to hardcode them.